python/tank/__init__.py

```diff
@@ -31,6 +31,47 @@
 
 import os
 import sys
+import uuid
+import inspect
+
+
+def __fix_tank_vendor():
```

Comment:
This fixes an issue where we use a tank_vendor from another core. This used to not be a big deal, but since we've added `six` to `tank_vendor`, we need to fix it now.